bamboo/docker-compose.yml

```diff
@@ -46,7 +46,7 @@ services:
     environment:
       SERVICES: "cloudformation,cloudwatch,cloudwatchlogs,dynamodb,iam,kinesis,kms,lambda,s3,secretsmanager,sns,sqs,stepfunctions,ssm,logs"
   build_env:
-    image: maven.earthdata.nasa.gov/cumulus:latest
+    image: ${bamboo_CUMULUS_BASE_IMAGE:-maven.earthdata.nasa.gov/cumulus:latest}
```

Comment:
Is the implication that we'd have a base image version for incompat image changes?

Reply:
Ah, I see there's a slack thread.